narwhal/primary/src/tests/core_tests.rs

```diff
@@ -557,6 +558,248 @@ async fn process_certificates() {
         3
     );
 }
+#[tokio::test(flavor = "current_thread", start_paused = true)]
```

Comment:
This is because I want the sleep statement I added to not actually take 5 seconds, but to simulate enough time for the submitted certificates to get added to the cert store.

Reply:
Sorry for coming back to this, but I want to make sure I understand it (haven't used the `start_paused` property before so I might miss things)...so you don't expect the sleep statement to actually take 5 seconds? My understanding is that the `pause` property in general pauses the underneath runtime clock - so will the `sleep` here behave differently than the expected?